pages/getting-started/writing-your-first-test.md

```diff
@@ -272,4 +272,108 @@ Leia sobre isso [aqui](https://docs.cypress.io/guides/tooling/IDE-integration.ht
+URL é o URL esperado. Podemos fazer isso procurando a URL e encadeando uma asserção a ela com `.should()`.
+
+Aqui está, algo parecido:
+
+```javascript
+describe('My First Test', () => {
+  it('clicking "type" navigates to a new url', () => {
+    cy.visit('https://example.cypress.io')
+
+    cy.contains('type').click()
+
+    // Should be on a new URL which includes '/commands/actions'
+    cy.url().should('include', '/commands/actions')
+  })
+})
+```
+
+#### Adicionando mais comandos e asserções
+
+Não estamos limitados a uma única interação e asserção em um determinado teste. Na verdade, muitas
+interações em um aplicativo podem exigir várias etapas e provavelmente alterarão o estado do aplicativo
+em mais de uma maneira.
+
+Podemos continuar as interações e asserções neste teste, adicionando outro encadeamento para interagir
+e verificar o comportamento dos elementos nesta nova página.
+
+Podemos usar `cy.get()` para selecionar um elemento baseado em uma classe CSS. Então, podemos usar o comando
+`.type()` para inserir texto no campo de entrada de texto selecionado. Por fim, podemos verificar que o valor
+do campo de entrada de texto reflete o texto que foi digitado com outro `.should()`.
+
+```javascript
+describe('My First Test', () => {
+  it('Gets, types and asserts', () => {
+    cy.visit('https://example.cypress.io')
+
+    cy.contains('type').click()
+
+    // Should be on a new URL which includes '/commands/actions'
+    cy.url().should('include', '/commands/actions')
+
+    // Get an input, type into it and verify that the value has been updated
+    cy.get('.action-email')
+      .type('fake@email.com')
+      .should('have.value', 'fake@email.com')
+  })
+})
+```
+
+E aí está: um breve teste no Cypress que visita uma página, encontra e clica em um link, verifica a URL e,
+em seguida, verifica o comportamento de um elemento na nova página. Se lermos em voz alta, pode soar como:
+
+1. Visitar: `https://example.cypress.io`
+2. Encontrar o elemento que contém `type`
+3. Clicar no elemento
+4. Pegar a URL
+5. Verificar que ela inclui `/commands/actions`
+6. Pegar o campo de entrada de texto com a classe `.action-email`
+7. Digitar `fake@email.com` no campo de entrada de texto
+8. Verificar que o campo de entrada de texto é preenchido com o novo valor
+
+Ou na sintaxe Dado que, Quando, Então:
+
+1. Dado que um usuário visita `https://example.cypress.io`
+2. Quando ele clica no link com label `type`
+3. E ele digita `fake@email.com` no campo de entrada de texto com a classe `.action-email`
+4. Então o URL deve incluir `/comandos/ações`
+5. E o campo de entrada de texto com a classe `.action-email` tem `fake@email.com` como valor
+
+E ei, este é um teste muito limpo! Não precisamos dizer nada sobre como as coisas funcionam,
+apenas que gostaríamos de verificar uma série particular de eventos e resultados.
+
+[Vídeo de exemplo](https://docs.cypress.io/img/snippets/first-test-assertions-30fps.mp4)
```

Comment:
Será que não conseguimos deixar o vídeo em exibição igual na doc oficial e nesse exemplo abaixo:

[![Visão Geral do Cypress Dashboard.](https://img.youtube.com/vi/ezp60FUnjGg/0.jpg)](https://youtu.be/ezp60FUnjGg)

Reply:
@pedrohyvo Quando pesquisei sobre isso lembro que era possível adicionar um vídeo embedded no markdown apenas com vídeos do youtube. Tem alguma idéia se é possível fazer com links fora do YT?

Reply:
@lucastfa-toptal eu acredito que fora do YouTube não temos como, infelizmente 😢